core-model/src/main/java/com/sequenceiq/cloudbreak/domain/projection/StackCrnView.java

```diff
@@ -0,0 +1,8 @@
+package com.sequenceiq.cloudbreak.domain.projection;
+
+public interface StackCrnView {
```

Comment:
I am not sure if this is needed, we have StackView and StackApiView also.

Reply:
The intention was to return the least amount of data from the database. Those views contain way more than we need here.

Reply:
It is clear, I am just not sure if there is significant difference between performances of querying these views. If it is, then it is reasonable.

Reply:
I think this interface can be easily refactored, deleted if we would need more data for this feature.

Reply:
alright